.gitignore

```diff
@@ -2,4 +2,6 @@ build/
 logs/
 nbproject/
 vendor/
+.idea/
 composer.lock
+composer.phar
```

Comment:
still, having it here does not harm anyone - it allows both usages of composer (installed globally or locally).
same for phpstrom files ignore...